feature-libs/product-configurator/tsconfig.lib.json

```diff
@@ -15,7 +15,13 @@
     "lib": ["es2020", "dom"],
     "paths": {
       "@spartacus/core": ["dist/core"],
-      "@spartacus/storefront": ["dist/storefrontlib"]
+      "@spartacus/storefront": ["dist/storefrontlib"],
+      "@spartacus/checkout/assets": ["dist/checkout/assets"],
+      "@spartacus/checkout/components": ["dist/checkout/components"],
+      "@spartacus/checkout/core": ["dist/checkout/core"],
+      "@spartacus/checkout": ["dist/checkout"],
+      "@spartacus/checkout/occ": ["dist/checkout/occ"],
+      "@spartacus/checkout/root": ["dist/checkout/root"]
     }
```

Comment:
Shouldn't we only have the relevant entry points (checkout/root and checkout/core) here?

Reply:
It's enforced by the pipeline.  If I remove the extra entry points in the coonfig, I get:
```
--- feature-libs/product-configurator/tsconfig.lib.json ------------------------
 ✖  Property "@spartacus/checkout/assets": ["dist/checkout/assets"] is missing in compilerOptions.paths.
 ✖  Property "@spartacus/checkout/components": ["dist/checkout/components"] is missing in compilerOptions.paths.
 ✖  Property "@spartacus/checkout/core": ["dist/checkout/core"] is missing in compilerOptions.paths.
 ✖  Property "@spartacus/checkout": ["dist/checkout"] is missing in compilerOptions.paths.
 ✖  Property "@spartacus/checkout/occ": ["dist/checkout/occ"] is missing in compilerOptions.paths.

```